pygmt/helpers/decorators.py

```diff
@@ -13,111 +13,94 @@
-            the viewpoint. Default is [180, 90]. Full documentation is at
-            :gmt-docs:`gmt.html#perspective-full`.
-        """,
-    "registration": """\
-        registration : str
-            ``[g|p]``
-            Force output grid to be gridline (g) or pixel (p) node registered.
-            Default is gridline (g).""",
-    "t": """\
-        transparency : float
-            Set transparency level, in [0-100] percent range.
-            Default is 0, i.e., opaque.
-            Only visible when PDF or raster format output is selected.
-            Only the PNG format selection adds a transparency layer
-            in the image (for further processing). """,
-    "x": """\
-        cores : int
-            ``[[-]n]``.
-            Limit the number of cores to be used in any OpenMP-enabled
-            multi-threaded algorithms. By default we try to use all available
-            cores. Set a number *n* to only use n cores (if too large it will
-            be truncated to the maximum cores available). Finally, give a
-            negative number *-n* to select (all - n) cores (or at least 1 if
-            n equals or exceeds all).
-            """,
+    "R": r"""region : str or list
+    *xmin/xmax/ymin/ymax*\ [**+r**\ ][**+u**\ *unit*].
+    Specify the region of interest. This is a required argument if this
+    is the first plot command.""",
+    "J": r"""projection : str
+    *projection*\ [*projection-specific arguments*\ ]\ *figure size*.
+    Select map projection. This is a required argument if this
+    is the first plot command.""",
+    "B": """frame : bool or str or list
+    Set map boundary frame and axes attributes.""",
+    "U": """timestamp : bool or str
+    Draw GMT time stamp logo on plot.""",
+    "CPT": """cmap : str
+    File name of a CPT file or ``C='color1,color2[,color3,...]'`` to
+    build a linear continuous CPT from those colors automatically.""",
+    "G": """color : str
+    Select color or pattern for filling of symbols or polygons. Default
+    is no fill.""",
+    "V": """verbose : str
+    Select verbosity level [Default is w], which modulates the messages
+    written to stderr. Choose among 7 levels of verbosity:
+        - **q** - Quiet, not even fatal error messages are produced
+        - **e** - Error messages only
+        - **w** - Warnings [Default]
+        - **t** - Timings (report runtimes for time-intensive algorthms);
+        - **i** - Informational messages (same as "verbose=True")
+        - **c** - Compatibility warnings
+        - **d** - Debugging messages""",
+    "W": """pen : str
+    Set pen attributes for lines or the outline of symbols.""",
+    "XY": r"""xshift : str
+    [**a**\|\ **c**\|\ **f**\|\ **r**\][*xshift*].
+    Shift plot origin in x-direction.
+
+    yshift : str
+    [**a**\|\ **c**\|\ **f**\|\ **r**\][*yshift*].
+    Shift plot origin in y-direction. Full documentation is at
+    :gmt-docs:`gmt.html#xy-full`.""",
```

Comment:
The **Y** option is not correctly rendered.
![image](https://user-images.githubusercontent.com/3974108/107437140-87f1b680-6afc-11eb-8d00-22bf4cf2c106.png)

Reply:
I tried adding blank lines to create line breaks in bc6f1e8, but that just caused tests to fail. Any idea how to fix this?